Electronics/PCB/AstroPlant_ExtensionBoard/astroplant-extension-board-mk0-6-1/README.md

```diff
@@ -0,0 +1,42 @@
+
+#If you want to generate output files, follow these steps: 
+
+First you'll need Kicad 7.0: https://www.kicad.org/download/ 
+Then open the project by executing the .kicad_pro file, you will then be able to see the different elements of the project such as the schematic and the pcb,
+If you want the gerber or drill map, open the pcb editor (.kicad_pcb) and go "File -> Fabrication output ->Gerbers/Drill/BOM".
+
```

Comment:
It would be useful to know what and why do we need to manufacture the board, for example:

```suggestion
To manufacture the PCB you need these files:

  - gerber - (what for)
  - .drl - (what for)
 ...
```
Also, can you double check bom location (here it's not in Outputs)
And the pkp / pos files are not included above.

Reply:
It's normal not to have the BOM in the output files, as it's not used to manufacture the board. 
The Pkp is just another file you have to generate like drill/gerber, i'll add it to the readme